city/crud.py

```diff
@@ -0,0 +1,59 @@
+from sqlalchemy.future import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from city import models, schemas
+
+
+async def post_city(db: AsyncSession, city_schema: schemas.CityCreate):
+    city = models.City(
+        name=city_schema.name,
+        additional_info=city_schema.additional_info
+    )
+    db.add(city)
+    await db.commit()
+    await db.refresh(city)
+    return city
+
+
+async def get_all_cities(db: AsyncSession) -> list[models.City]:
```

Comment:
Consider using `list[models.City]` as a return type hint for `get_all_cities` to ensure compatibility with Python versions prior to 3.9. Alternatively, you can import `List` from `typing` and use `List[models.City]`.